packages/client/src/__tests__/integration/happy/rejectNotFound/test.ts

```diff
@@ -49,21 +49,31 @@ for (const constructorKey of Object.keys(cases.constructor)) {
     for (const valueKey of Object.keys(currentMethod)) {
       const value = currentMethod[valueKey]
       test(`rejectOnNotFound | constructor=${constructorKey} | ${method}=${value}`, async () => {
-        // It should fail or not
-        expect.assertions(1)
         const PrismaClient = await getTestClient()
         const prisma = new PrismaClient({
           rejectOnNotFound: constructor,
         })
 
-        // Test Rejection
-        try {
+        // This function name is important cause we're searching
+        // for it's name in the stack bellow
+        const testRejectionOnNotFound = async () => {
           const r = await prisma.user[method]({
             where: { id: 'none' },
             rejectOnNotFound: value,
           })
-          expect(r).toMatchSnapshot()
+
+          // If it got to here, then no error was thrown
+          // so we check the value if it's equal to null
+          expect(r).toBeNull()
+        }
+
+        try {
+          await testRejectionOnNotFound()
         } catch (error) {
+          const { message, stack } = error
+          expect(stack).toBeDefined()
+          expect(message).toBeDefined()
+
           expect(error).toMatchSnapshot()
```

Comment:
```suggestion
          expect(error).toMatchSnapshot()
          expect(stack).toContain('at testRejectionOnNotFound')
```

Reply:
https://github.com/prisma/prisma/pull/10645#discussion_r768650783

Reply:
(FWIW, I would also be fine with just checking that the stack is not undefined, but then it kinda defeats the purpose of all other changes, and they all could be reverted and replaced with a one line addition. Since we already have them anyway, we may assert something useful about the stack trace as well).

Reply:
oh I've removed it by mistake, thanks for noticing